development/file_formats/tscn.rst

```diff
@@ -1,84 +1,87 @@
 TSCN file format
 ================
 
-A :code:`.tscn` File format is the "Text SCeNe" file format and represents
-a single scene-tree inside Godot. TSCN files have the advantage of being
-nearly human-readable and easy for version control systems to manage. During
-import the TSCN files are compiled into binary :code:`.scn` files stored
-inside the .import folder. This reduces the data size and speed up loading.
+The TSCN (text scene) file represents a single scene tree inside
+Godot. TSCN files have the advantage of being mostly human-readable and easy for
+version control systems to manage. During import, TSCN files are compiled into
+binary ``.scn`` files stored inside the .import folder. This reduces the data
+size and speeds up loading.
 
-The :code:`.escn` file format is identical to the TSCN file format, but is used to
+The ``.escn`` file format is identical to the TSCN file format, but is used to
```

Comment:
Why `.escn` here but `TSCN` above?

Reply:
I don't understand what you meant here. Could you clarify, please?

Reply:
"The TSCN (text scene) file format" vs "The `.escn` file format", one has the format capitalized, the other has a dot and then the format name.